cli/cmd/getCluster.go

```diff
@@ -101,7 +101,12 @@ func Printer(i int) {
 	if err != nil {
 		panic(fmt.Errorf("JSON Convertion failed"))
 	}
-	printUtil(arr)
+	if len(toBePrinted) == 0 {
+		fmt.Println("No clusters found")
+	} else {
+		printUtil(arr)
+
+	}
```

Comment:
when the json.marshal converts a empty json it returns nil does that compares to len() == 0 ?

Reply:
@bishal7679 you should check for nil as its of type []byte

Reply:
@bishal7679 have you tested this in local system?